internal/db/schema/migrations/oss/postgres/99/01_split_worker_tag_table.up.sql

```diff
@@ -0,0 +1,88 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+-- Create new tables for worker config and worker api tags
+create table server_worker_config_tag(
+  worker_id wt_public_id
+    constraint server_worker_fkey
+      references server_worker (public_id)
+      on delete cascade
+      on update cascade,
+  key wt_tagpair,
+  value wt_tagpair,
+  primary key (worker_id, key, value)
+);
+comment on table server_worker_config_tag is
+  'server_worker_config_tag is a table where each row represents a worker config tag.';
+
+create table server_worker_api_tag(
+  worker_id wt_public_id
+    constraint server_worker_fkey
+      references server_worker (public_id)
+      on delete cascade
+      on update cascade,
+  key wt_tagpair,
+  value wt_tagpair,
+  primary key (worker_id, key, value)
+);
+comment on table server_worker_api_tag is
+  'server_worker_api_tag is a table where each row represents a worker api tag.';
+
+-- Migrate from server_worker_tag to the new tables
+ insert into server_worker_config_tag
+             (worker_id, key, value)
+      select worker_id, key, value
+        from server_worker_tag
+       where source = 'configuration';
+
+ insert into server_worker_api_tag
+             (worker_id, key, value)
+      select worker_id, key, value
+        from server_worker_tag
+       where source = 'api';
+
+
+drop view server_worker_aggregate;
+-- Replaces view created in 86/01_server_worker_local_storage_state.up.sql to use the disparate worker tag tables
+-- View also switches to using json_agg to build the tags for consumption
+create view server_worker_aggregate as
```

Comment:
I assume the view is just temporary while we move to all the individual queries? If so perhaps a comment would be helpful

Reply:
it is temporary- I'll add a comment, thanks!